plugins/lookup/passwordstore.py

```diff
@@ -60,6 +60,10 @@
         description: use alphanumeric characters.
         type: bool
         default: 'no'
+      ignore_missing:
+        description: Do not report error upon non-existent password file
```

Comment:
It's clearer this way IMO.
```suggestion
        description: Do not fail upon non-existent password file.
```